stable/traefik/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.7.0
-appVersion: 1.3.1
+version: 1.8.0
```

Comment:
Presumably, the delta between Traefik `1.3.1` and `1.3.3` is only bug fixes. If that's the case, the net effect on the chart is also only bug fixes. So shouldn't the new chart version be `1.7.1`?